tests/tests/cluster/test-command-palette.spec.js

```diff
@@ -214,14 +214,6 @@ context('Test Command Palette navigation', () => {
   it('Disables Command Palette if a modal is present', () => {
     openCommandPalette();
 
-    cy.getModalIframeBody()
-      .find('[aria-label=command-palette-search]')
-      .should('not.exist');
-
-    cy.getIframeBody().contains('Cluster Details');
```

Comment:
why did we remove it?

Reply:
That part of the test depended on the previous one. We opened the `Preferences` modal, but we didn't close it, which in my opinion is a bit wrong because tests should be independent. The new Preferences modal behaves like any other `Dialog` component, so there's no need to have a special test case for that. That's why I removed that.